charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml

```diff
@@ -295,6 +295,20 @@ spec:
           readOnly: true
           {{- end }}
         {{- end }}
+      {{- if .Values.sni.enabled }}
+      - name: apiserver-proxy-pod-mutator
+        image: {{ index .Values.images "apiserver-proxy-pod-webhook" }}
+        args:
+        - --apiserver-fqdn={{ .Values.sni.apiserverFQDN }}
+        - --host=localhost
+        - --cert-dir=/srv/kubernetes/apiserver
+        - --cert-name=kube-apiserver.crt
+        - --key-name=kube-apiserver.key
+        - --port=9443
+        volumeMounts:
```

Comment:
Should we add resource requests and limits for this container?

Reply:
I'll rather wait and gather some metrics from various workloads on different clusters before adding such requests / limits.